pontoon/sync/tasks.py

```diff
@@ -125,15 +125,10 @@ def sync_project(
         locale=locale,
         no_pull=no_pull,
         no_commit=no_commit,
-        full_scan=force
+        full_scan=force,
+        new_entities=source_changes.get('new_entities')
```

Comment:
Could you instead pass `source_changes.get('new_entities')` as a positional argument, consistently with other `source_changes` arguments?